content/en/docs/setup/getting-started/index.md

```diff
@@ -52,6 +52,14 @@ Follow these steps to get started with Istio:
     $ cd istio-{{< istio_full_version >}}
     {{< /text >}}
 
+    The installation directory contains:
+
+    - Installation YAML files for Kubernetes in `install/kubernetes`
+    - Sample applications in `samples/`
+    - The [`istioctl`](/docs/reference/commands/istioctl) client binary in the
+      `bin/` directory. `istioctl` is used if you choose to manually inject
```

Comment:
`istioctl` is also used to get at the dashboard and install Istio. I would be inclined to simply remove the `istioctl is used ...` instead of adding a list of items.

Reply:
fixed